Scenes/SceneBase.gd

```diff
@@ -50,6 +51,9 @@ func initScene(args = []):
 	_initScene(args)
 	_reactInit()
 	
+	if(args.has("pc")):
+		sceneHasPC = true
```

Comment:
nah, that's not how you do it. Your best would be to make the `isSexSceneWithPC` return false by default and then override that behavior for the procedural sex scene and other scenes that you need